sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -315,6 +314,40 @@ def test_dataframe_drop_ignore_errors(self):
     self._run_test(
         lambda df: df.drop(index='cow', columns='small', errors='ignore'), df)
 
+  def test_groupby_apply(self):
+    df = GROUPBY_DF
+
+    def median_sum_fn(x):
+      return (x.foo + x.bar).median()
+
+    describe = lambda df: df.describe()
```

Comment:
Why not use `DataFrame.describe`?

Reply:
Yeah this is the same as `DataFrame.describe`, it's just a convenient function to test the case where the applied fn returns a Series.

Reply:
er actually this is different, because it's producing summary statistics per group. It is the same as [`DataFrameGroupBy.describe`](https://pandas.pydata.org/docs/reference/api/pandas.core.groupby.DataFrameGroupBy.describe.html) though.